### PR TITLE
design updates

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

- updates icons, colors, sizes, spaces
- updates responsiveness of filer navigator head
- compared styles with pagetree

https://github.com/divio/django-cms/pull/5167
https://github.com/divio/djangocms-admin-style/pull/322

### CODE REVIEW

File: filer/private/sass/components/_filter-files.scss [hunk 1]
```diff
@@ -48,6 +41,8 @@
         }
         &.open + .dropdown-menu-checkboxes {
             display: block;
+            width: 100%;
+            transition: width .2s;
```
Comment (FinalAngel) [inline filer/private/sass/components/_filter-files.scss]: ideally also add this as a variable like: https://github.com/aldryn/aldryn-boilerplate-bootstrap3/blob/master/private/sass/settings/_custom.scss#L7

File: filer/private/sass/components/_filter-files.scss [hunk 2]
```diff
@@ -89,8 +84,8 @@
         margin: 0;
         padding: 0 35px 0 10px !important;
         outline: none;
-
         appearance: none;
+        transition: width .2s;
```
Comment (FinalAngel) [inline filer/private/sass/components/_filter-files.scss]: same here

File: filer/templates/admin/filer/folder/directory_table.html [hunk 3]
```diff
@@ -209,5 +210,18 @@
                 {% trans "next" %}
             </a>
         {% endif %}
+        <div class="actions">
+            {% if actions_selection_counter %}
+                <script type="text/javascript">var _actions_icnt="{{ paginated_items.object_list|length|default:"0" }}";</script>
```
Comment (FinalAngel) [inline filer/templates/admin/filer/folder/directory_table.html]: remove typr

File: filer/templates/admin/filer/folder/directory_table.html [hunk 4]
```diff
@@ -209,5 +210,18 @@
...
+                <span class="action-counter">{{ selection_note }}</span>
+                {% if paginator.count != paginated_items.object_list|length %}
+                    <span class="all">{{ selection_note_all }}</span>
+                    <span class="question">
+                        <a href="javascript:;" title="{% trans "Click here to select the objects across all pages" %}">{% blocktrans with paginator.count as total_count %}Select all {{ total_count }}{% endblocktrans %}</a>
```
Comment (FinalAngel) [inline filer/templates/admin/filer/folder/directory_table.html]: why an empty javascript:;?

File: filer/templates/admin/filer/folder/directory_table.html [hunk 5]
```diff
@@ -209,5 +210,18 @@
...
+                    </span>
+                    <span class="clear"><a href="javascript:;">{% trans "Clear selection" %}</a></span>
```
Comment (FinalAngel) [inline filer/templates/admin/filer/folder/directory_table.html]: same here?

File: filer/private/sass/components/_navigator.scss [hunk 6]
```diff
@@ -22,6 +22,13 @@ body {
     pointer-events: none;
     z-index: 100;
 }
+.table-head-shadow {
+    position: fixed;
+    clip: rect(0,99999px,99999px,-2px);
+    box-shadow: 0 2px 5px -2px rgba(0,0,0,0.2);
```
Comment (vxsx) [inline filer/private/sass/components/_navigator.scss]: `(black, 0.2)`

File: filer/private/sass/mixins/_custom.scss [hunk 7]
```diff
@@ -69,7 +69,7 @@
         &:focus
         &:active {
             background-color: rgba($background, 0.4) important($important);
-            border-color: rgba($border, 0.4) important($important);
+            border-color: rgba($border, 0) important($important);
```
Comment (vxsx) [inline filer/private/sass/mixins/_custom.scss]: why? will cause inconsistency with cms

File: filer/templates/admin/filer/folder/directory_table.html (hunk 4 above)
Comment (vxsx) [inline filer/templates/admin/filer/folder/directory_table.html]: essentially same as `javascript:void(0);` (would rather use full version here though)

File: filer/private/sass/mixins/_custom.scss (hunk 7 above)
Comment (lory87) [inline filer/private/sass/mixins/_custom.scss]: I have to remove design wise border on disabled buttons

**Comment by vxsx** [comment]
👍

**Pull request merged**